apps/dashboard/app/models/batch_connect/session.rb

```diff
@@ -136,7 +136,7 @@ def script_template(context = {})
     # @return [Boolean] whether saved successfully
     def save(app:, context:, format: nil)
       self.id         = SecureRandom.uuid
-      self.cluster_id = app.cluster_id
+      self.cluster_id = context.cluster
```

Comment:
I forget, does this mean that cluster can be changed in the submit.yml? So the decision to which cluster to submit to could be completely handled here?

Reply:
submit.yml is templated _after_ this assignment, so you'd get the same behaviour as before. You're able to submit to wherever but since `self.cluster_id` was saved, that's the cluster we'll query for info.

Reply:
Addressed in https://github.com/OSC/ondemand/issues/529.